builder/amazon/ebs/step_create_ami.go

```diff
@@ -48,7 +51,22 @@ func (s *stepCreateAMI) Run(ctx context.Context, state multistep.StateBag) multi
 		BlockDeviceMappings: config.AMIMappings.BuildEC2BlockDeviceMappings(),
 	}
 
-	createResp, err := ec2conn.CreateImage(createOpts)
+	var createResp *ec2.CreateImageOutput
+	var err error
+
+	err = retry.Config{
+		Tries: 11,
+		ShouldRetry: func(err error) bool {
+			if err, ok := err.(awserr.Error); ok {
+				return err.Code() == "InvalidParameterValue"
```

Comment:
`InvalidParameterValue` sounds like a non-retriable error but reading the logs is more :

`InvalidParameterValue: Invalid value 'i-05348a0d76476e460' for instanceId. Instance is not in state 'running' or 'stopped'`

LGTM !

Reply:
Ah we could use:

```go
			if isAWSErr(err, "InvalidParameterValue", "Instance is not in state") {
				return true
			}
```

Reply:
Yeah, that's probably better. I was being lazy and didn't want to convert isAwsErr to an exported value for this, but it's the right call.